### ISSUE TITLE
Pro controller usb connectivity ?

### ISSUE THREAD

**Issue by remipache** [CTCaer/jc_toolkit#24]
I was wondering if pro controller connected via usb support is a planned feature for jc_toolkit.
I don't have any idea about how hard it would be to implement, but i thing it could be a really nice and useful feature for those (like me) who don't have bluetooth in their computers

**Comment by NHurst99** [comment]
I agree, although, my computer does have Bluetooth, I have an adapter that lets me use Xbox/ps controllers on the switch, but it shows it as a white controller with white buttons and I would love to be able to change the colors

**Comment by Heartbeat-Heartbreak** [comment]
For those who need it: you can do it through steam. 
Connect your PC and Pro with usb, open steam, if everything goes well the home button of your Pro will light up, then open jc_toolkit and it will recognize your Pro.
